packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -23,10 +41,40 @@ export default function ErrorComponentPreview(props: {
   }, []);
 
   const rect = containerRef.current?.getBoundingClientRect();
-  const tooltipPosition = rect && {
-    x: (rect.right + rect.left) / 2,
-    y: rect.top - 40,
-  };
+  const tooltipPosition = useMemo(() => {
+    return (
+      rect && {
+        x:
+          (rect.right + rect.left) / 2 + window.innerWidth / 4 - window.scrollX,
+        y: rect.top + 25 - window.scrollY,
+      }
+    );
+  }, [rect]);
+
+  document
+    .querySelector("iframe")
+    ?.contentDocument?.addEventListener("scroll", () => {
+      setTooltipOpen(false);
+    });
+
+  document.addEventListener("scroll", () => {
```

Comment:
nit: I slightly prefer the simplified syntax for the callback (`() => setTooltipOpen(false)`) here and above. it makes it more clear to me that this is the only action occurring